fabfile.py

```diff
@@ -397,23 +403,23 @@ def run_backup(self, email=False):
         sudo('zip /tmp/{0}-{1}.backup /tmp/odoo-{0}.tar /tmp/{0}.sql'
              .format(self.instance, date))
 
-        #save in instance/backup folder
-        backup = os.path.isdir("backup")
-        if not backup:
-            os.system("mkdir backup")
-        os.system('scp -r applejuice.sunflowerweb.nl:/tmp/{0}.backup backup/{0}.backup'
+        os.system('scp -r applejuice.sunflowerweb.nl:/tmp/{0}* /mnt/volume-fra1-01/sfit-backup'
             .format(self.instance))
 
         #remove files from /tmp
         sudo('rm -f /tmp/{0}.sql /tmp/odoo-{0}.tar tmp/{0}-{1}.backup'.format(self.instance, date))
 
         #send email after successful backup
-        msg = "Backup for {} at {} successful.".format(self.instance, date)
-        sudo('echo {} | mail {}'.format(msg, email))
+        msg = "Backup for {0} at {1} successful.".format(self.instance, date)
+        script = "echo '{1}' | mail -s 'Backup for {0} instance.' {2}".format(
+            self.instance, msg, email)
+        p = subprocess.Popen(
+            script, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+        )
```

Comment:
Hm, you are still using `echo bla | mail bla@bla`, and then also two times in the code. Could you still use [sendmail](https://stackoverflow.com/questions/73781/sending-mail-via-sendmail-from-python) and then abstract it into a subfunction, so that you can just call it twice?